uvloop/_testbase.py

```diff
@@ -67,6 +67,9 @@ def new_loop(self):
     def mock_pattern(self, str):
         return MockPattern(str)
 
+    def has_start_serving(self):
+        return not (self.is_asyncio_loop() and sys.version_info[:2] in [(3, 5), (3, 6)])
```

Comment:
Everything looks great. One last nit: please make sure all lines are shorter than 80 chars.

Reply:
Sure thing. I wasn't too sure as the limit doesn't seem to be 100% enforced. Maybe we should get CI to enforce this ?

Reply:
Sure: https://github.com/MagicStack/uvloop/pull/229 :)

Reply:
Speaking of which, let's land https://github.com/MagicStack/uvloop/pull/229 first and then rebase this one.